src/reducers/combined-updaters.js

```diff
@@ -78,10 +81,47 @@ const combinedUpdaters = null;
 
 export const isValidConfig = config =>
   isPlainObject(config) && isPlainObject(config.config) && config.version;
+
 export const defaultAddDataToMapOptions = {
   centerMap: true,
   keepExistingConfig: false
 };
+
+const identity = state => state;
+
+/* eslint-disable-next-line no-unused-vars */
```

Comment:
is this necessary?

Reply:
log is a useful function for debugging

Reply:
no I meant if the  __/* eslint-disable-next-line no-unused-vars */__ line is necessary

Reply:
yes, because log function is currently not used, it's for debugging